client/components/date-range/index.js

```diff
@@ -176,7 +180,7 @@ export class DateRange extends Component {
 	isValidDate( date ) {
 		const { firstSelectableDate, lastSelectableDate } = this.props;
 
-		const epoch = this.props.moment( '01/01/1970', this.getLocaleDateFormat() );
+		const epoch = this.props.moment( '01/01/1970' );
```

Comment:
In the interests of changing as little code as possible, we could hardcode this to

```suggestion
		const epoch = this.props.moment( '01/01/1970', this.props.moment.localeData().longDateFormat( 'L' ) );
```